packages/core/src/code-evaluation/rebuildGeometryCli.js

```diff
@@ -1,34 +1,19 @@
 import path from 'path'
-import { createRequire } from 'module'
 
 import { toArray } from '@jscad/array-utils'
 
-import { requireDesignFromModule } from '../code-loading/requireDesignFromModule.js'
 import { getParameterDefinitionsAndValues } from '../parameters/index.js'
-import { makeWebRequire } from '../code-loading/webRequire.js'
 
 export const rebuildGeometryCli = async (data) => {
   const defaults = {
     apiMainPath: '@jscad/modeling'
   }
-  let { apiMainPath, mainPath, parameterValues, useFakeFs } = Object.assign({}, defaults, data)
-  // we need to update the source for our module
-  let requireFn = createRequire(import.meta.url)
+  let { mainPath, parameterValues, useFakeFs } = Object.assign({}, defaults, data)
 
   // source came from conversion, i.e. not from file system
   if (useFakeFs) {
     const pathParts = path.parse(mainPath)
-    const fakeName = `${pathParts.name}.js`
     const fakePath = `/${pathParts.name}.js`
-    const filesAndFolders = [
```

Comment:
hmm... i think this is used when converting from STL to JS. maybe we don't have a test case?

Reply:
It was used by `makeWebRequire` but that got replaced with a call to `await import(mainPath)`, so `filesAndFolders` became unused.

I reverted this change though, since I think we will need to revisit the code loading.